<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>724</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>724</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartHouse Extreme</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="fireButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать пожар</string>
    </property>
   </widget>
   <widget class="QLabel" name="fireLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/images/fire1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="sprinkLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/images/sprink1.png</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="intruder">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/images/intruder1.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="intruderButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить нарушителя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeIntruderButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Убрать нарушителя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/images/house.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/images/gsm.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>241</width>
      <height>123</height>
     </rect>
    </property>
    <property name="title">
     <string>Системы дома</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="fireExLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="pictures.qrc">:/images/butOff.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="onFButton">
         <property name="text">
          <string>Вкл. пож. трев.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="offFButton">
         <property name="text">
          <string>Выкл. пож. трев</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="tresPasLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="pictures.qrc">:/images/butOff.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="onTresButton">
         <property name="text">
          <string>Вкл. охрану</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="offTresButton">
         <property name="text">
          <string>Выкл. охрану</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Валяев, ГУАП 4242</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>fireButton</zorder>
   <zorder>textEdit</zorder>
   <zorder>intruderButton</zorder>
   <zorder>removeIntruderButton</zorder>
   <zorder>label</zorder>
   <zorder>sprinkLabel</zorder>
   <zorder>fireLabel</zorder>
   <zorder>intruder</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
